src/diffusers/models/attention_processor.py

```diff
@@ -441,12 +448,22 @@ class XFormersAttnProcessor:
     def __init__(self, attention_op: Optional[Callable] = None):
         self.attention_op = attention_op
 
-    def __call__(self, attn: Attention, hidden_states, encoder_hidden_states=None, attention_mask=None):
-        batch_size, sequence_length, _ = (
+    def __call__(
+        self,
+        attn: Attention,
+        hidden_states: FloatTensor,
```

Comment:
```suggestion
        hidden_states: torch.FloatTensor,
```
Just to align with style format

Reply:
restored namespace qualifier on all uses of *Tensor symbols